packages/orchestration/src/exos/local-orchestration-account.js

```diff
@@ -155,27 +153,28 @@ export const prepareLocalOrchestrationAccountKit = (
          * @param {string} validatorAddress
          * @param {Amount<'nat'>} ertpAmount
          */
-        async Delegate(validatorAddress, ertpAmount) {
+        Delegate(validatorAddress, ertpAmount) {
           trace('Delegate', validatorAddress, ertpAmount);
 
-          return zcf.makeInvitation(async seat => {
-            // TODO should it allow delegating more BLD?
+          return zcf.makeInvitation(seat => {
             seat.exit();
-            return this.facets.holder.delegate(validatorAddress, ertpAmount);
+            return watch(
```

Comment:
note to self: is a test for this change included in this PR?

If not, what's our plan for testing it?

Reply:
the 1st line of the commit message exceeds conventional norms

I suggest changing `feat(orchestration):` to `chore:`. The feature that I would expect to see in release notes is local-orchestration-account.